tiledb/sm/array_schema/array_schema_evolution.h

```diff
@@ -36,8 +36,9 @@
 
 #include <unordered_map>
 #include <unordered_set>
-
 #include "tiledb/common/common.h"
+#include "tiledb/common/memory_tracker.h"
```

Comment:
Don't include memory_tracker.h in header files, use a forward declaration for MemoryTracker instead.

Reply:
Got it!